ngraph_creator/operations/src/Add.cpp

```diff
@@ -14,12 +14,6 @@ Add::Add(int operationIndex) : OperationsBase(operationIndex) {
 bool Add::validate() {
     ALOGV("%s PASSED", __func__);
 
-    const auto& activationIndex = sModelInfo->getOperationInput(mNnapiOperationIndex, 1);
-    if (!sModelInfo->isOperandLifeTimeConst(activationIndex)) {
-        ALOGE("%s Only Constant supported for specifying Activation", __func__);
```

Comment:
More descriptive error msg to capture why this validation check exists.

Reply:
@anikulk Done